app/utils/whitelabel.ts

```diff
@@ -1,31 +1,89 @@
 import config from '../config';
+import icon from '../assets';
+import styles from '../components/ui/Navigation/Navigation.module.scss';
 
-/** Whether we should display the SF Service Guide branded version of the site */
-const isSFFamiliesSite = () => window.location.host.indexOf(config.SFFAMILIES_DOMAIN) > -1;
+// Include new white label here
+type WhiteLabelSiteKey = 'default' | 'SFServiceGuide' | 'SFFamilies';
 
-const isSFServiceGuideSite = () => !isSFFamiliesSite()
-  && window.location.host.indexOf(config.MOHCD_DOMAIN) > -1;
+interface WhiteLabelSite {
+  title: string;
+  siteUrl: string;
+  appImages: {
+    background: string;
+     // For appImages, logoLarge and logoSmall are usually different between each white label
+    logoLarge: string;
+    logoSmall: string;
+    algolia: string;
+    mohcdSeal: string;
+    icon: any;
+  },
+  styles?: any
+}
 
-const getSiteTitle = () => {
-  if (isSFServiceGuideSite()) {
-    return 'SF Service Guide';
-  }
-  if (isSFFamiliesSite()) {
-    return 'SF Families';
-  }
-  return 'AskDarcel';
+// Read only to force developer to modify configurations here, disallow changes at compile time
+const configurations: Partial<Record<Readonly<WhiteLabelSiteKey>, Readonly<WhiteLabelSite>>> = {};
+
+// Include a domain in config.js
+function determineWhiteLabelSite(): WhiteLabelSiteKey {
+  if (config.SFFAMILIES_DOMAIN === window.location.host) return 'SFFamilies';
+  if (config.MOHCD_DOMAIN === window.location.host) return 'SFServiceGuide';
+  return 'default';
+}
+
+const configKey = determineWhiteLabelSite();
+
+// Specify what is viewed in each white label
+configurations.SFFamilies = {
+  title: 'SF Families',
+  siteUrl: 'https://sffamilies.sfserviceguide.org/',
+  appImages: {
+    background: BackgroundImage,
+    logoLarge: SFFamiliesLogo,
+    logoSmall: SFFamiliesLogo,
+    algolia: SearchByAlgoliaImage,
+    mohcdSeal: SFSeal,
+    icon,
+  },
+  styles: {
+    siteNavSFFamilies: styles.siteNavSFFamilies,
+    navLogoSFFamilies: styles.navLogoSFFamilies,
+  },
 };
 
-const getSiteUrl = () => {
-  if (isSFServiceGuideSite()) {
-    return 'https://sfserviceguide.org';
-  }
-  if (isSFFamiliesSite()) {
-    return 'https://sffamilies.sfserviceguide.org/';
-  }
-  return 'https://askdarcel.org';
+configurations.SFServiceGuide = {
+  title: 'SF Service Guide',
+  siteUrl: 'https://sfserviceguide.org',
+  appImages: {
+    background: BackgroundImage,
+    logoLarge: SFServiceLogo,
+    logoSmall: SFServiceLogo,
+    algolia: SearchByAlgoliaImage,
+    mohcdSeal: SFSeal,
+    icon,
+  },
 };
 
-export {
-  isSFServiceGuideSite, isSFFamiliesSite, getSiteTitle, getSiteUrl,
+configurations.default = {
+  title: 'AskDarcel',
+  siteUrl: 'https://askdarcel.org',
+  appImages: {
+    background: BackgroundImage,
+    logoLarge: AskDarcelImage,
+    logoSmall: AskDarcelImage,
+    algolia: SearchByAlgoliaImage,
+    mohcdSeal: SFSeal,
+    icon,
+  },
 };
+
+// Disallow changes at run time
```

Comment:
Also I want to make sure none of the objects are modified at run time to also encourage development in this file

Reply:
Similar to my other PR comment, I don't think this code comment is really saying anything that isn't already obvious from the code just below it. Could you remove it?